tests/nn/util_test.py

```diff
@@ -613,3 +613,16 @@ def test_remove_sentence_boundaries(self):
                              [1, 1, 1],
                              [1, 1, 0]]))).long()
         assert (new_mask.data.numpy() == expected_new_mask.data.numpy()).all()
+
+    def test_add_positional_features(self):
+        # This is hard to test, so we check that we get the same result as the
+        # original tensorflow implementation:
+        # https://github.com/tensorflow/tensor2tensor/blob/master/tensor2tensor/layers/common_attention.py#L270
+        tensor2tensor_result = numpy.asarray([[0.00000000e+00, 0.00000000e+00, 1.00000000e+00, 1.00000000e+00],
+                                              [8.41470957e-01, 9.99999902e-05, 5.40302277e-01, 1.00000000e+00],
+                                              [9.09297407e-01, 1.99999980e-04, -4.16146845e-01, 1.00000000e+00]])
+
+        tensor = Variable(torch.zeros([2, 3, 4]))
```

Comment:
Can you add a test here with `[2, 3, 5]`, so we get a test with an odd hidden dim?